docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+---
```

Comment:
Don't forget to include the document versions strings (markdown version and Google Doc version).